make-compose.js

```diff
@@ -59,15 +90,22 @@ const generateConfig = () => {
         depends_on: ["db", "wordpress"],
         image: "wordpress_installer:latest",
         environment: {
-          WP_HOST_NAME: `${config.WP_HOST_NAME}:${config.WP_PORT}`,
+          WP_HOST_NAME: wpHostName,
           WORDPRESS_DB_HOST: "db",
           WORDPRESS_DB_USER: "root",
           WORDPRESS_DB_PASSWORD: "root",
           WORDPRESS_DB_NAME: "wp_db",
         },
         user: "xfs",
         command: `/usr/local/bin/${INSTALL_SCRIPT_FILENAME}`,
-        volumes: ["wp_data:/var/www/html"],
+        volumes: [
+          ...mappings,
```

Comment:
Add the mappings here to fix the issue where the cli doesn't see the plugins (this correctly enables any that are already available).